### ISSUE TITLE
Distinct Pod Items

### ISSUE THREAD

**Issue by geshergalicia** [pods-framework/pods#3425]
It would be awesome if there were a way to force a Pod to only allow distinct items, i.e. it can't have two items with the same name. Alternatively it could be built so that a field within the Pod must be unique when compared to all other items in the Pod. This would allow us to build limited one-to-one relationship pod items per user.

**Comment by quasel** [comment]
well from https://wordpress.org/support/topic/unique-field/ - Unique fields are only available for Pods using table-based storage, we are working on implementing the ‘Unique’ feature for meta-based Pods soon.   - still plans for it @sc0ttkclark ? else we can close this issue ^^

**Comment by sc0ttkclark** [comment]
Unique fields are from the Pods 1.x Advanced Content Types, I can look at the code and see how far we got into enabling them for non-ACT and meta-based pods though.

**Comment by sc0ttkclark** [comment]
Yes, this is only currently available for Advanced Content Types as well as any table-based pod you create.

To enable Table-based storage for Pods, go to Pods Admin > Components and activate it there.

You'll also find Advanced Content Types there if you need something more bare-bones than Custom Post Types, but I wouldn't recommend it unless you want to be writing queries and URL rewrites.

**Comment by sc0ttkclark** [comment]
Further, this is not available for object relationships currently.

**Comment by JoryHogeveen** [comment]
@sc0ttkclark Status on this issue?
